docs/design/ros_workspace.md

```diff
@@ -38,7 +38,10 @@ vehicle platform in use.
 
 First, [metapackages](https://wiki.ros.org/Metapackages) are a new-ish ROS construct which helps define the build dependencies for a specific package. Essentially, a metapackage has no nodes or code. It is an empty package except for a `package.xml` and `CMakeLists.txt` file which define build dependencies. These build dependencies can then be used to directly build nodes/packages for a specific vehicle platform by only using `colcon build` to build that package.
 
-For instance, if a certain vehicle requires packages named `camera_driver`, `lidar_driver`, `perception`, `control`, and `actuation`, you can specify all these packages as `<build_depend>` in the metapackage. When `colcon build --packages-select <metapackage>` is run, the `<build_depend>` packages are automatically built.
+For instance, if a certain vehicle requires packages named `camera_driver`, `lidar_driver`, `perception`, `control`, and `actuation`, you can specify all these packages as `<exec_depend>` in the metapackage. When `colcon build --packages-up-to <metapackage>` is run, the `<exec_depend>` packages are automatically built.
+
+> [!NOTE]
```

Comment:
add a note somewhere that these are generic packages that we don't actually have yet, otherwise may be a little missleading?